From 6a04c0002fa4dd8085d96ce32ae858657763b3b7 Mon Sep 17 00:00:00 2001
From: Moritz Barsnick <moritz.barsnick@chargebyte.com>
Date: Mon, 8 Jun 2026 08:06:51 +0200
Subject: [PATCH] Restrict git describe tag search to standard patterns

This filters out custom tags which would appear in the version string.

Signed-off-by: Moritz Barsnick <moritz.barsnick@chargebyte.com>
---
 version-information.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version-information.cmake b/version-information.cmake
index 4a16670..5a75e15 100644
--- a/version-information.cmake
+++ b/version-information.cmake
@@ -10,7 +10,7 @@ function (evc_generate_version_information)
     if(GIT_FOUND)
         execute_process(
             COMMAND
-                "${GIT_EXECUTABLE}" describe --dirty --always --tags
+                "${GIT_EXECUTABLE}" describe --dirty --always --tags --match "2[0-9][0-9][0-9].[0-9]*.[0-9]*"
             WORKING_DIRECTORY
                 "${PROJECT_SOURCE_DIR}"
             RESULT_VARIABLE
-- 
2.34.1

